Tasks/RestoreCacheV0/task.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id": "D53CCAB4-555E-4494-9D06-11DB043FB4A9",
+    "name": "RestoreCache",
+    "friendlyName": "Restore Cache",
+    "description": "Downloads an associated cache.",
+    "helpUrl": "",
+    "helpMarkDown": "Downloads an associated cache to a local path.",
+    "category": "Utility",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 0
+    },
+    "groups": [],
+    "demands": [],
+    "preview": true,
+    "minimumAgentVersion": "2.140.1",
```

Comment:
Should the min Agent version just be 2.140.1? or higher than that

Reply:
Aah, I forgot to do a git push for this. Thanks.